A21-service-config-error-handling.md

```diff
@@ -44,8 +44,11 @@ if the chosen service config is found invalid while the other service configs
 were valid, the gRPC config would still be considered invalid as a whole.
 2. The client validates the choice based on the criteria outlined in the
 section below.
-3. If the service config is valid, the client uses the configuration provided.
-4. If the service config is invalid, the clients should reject the received
+3. If no service config is found, the client falls back on the default service
```

Comment:
Shouldn't this be (2) and (2) be renamed to (3)?

Reply:
You mean reordering the two? Will do

Reply:
Yeah, reorder it between the current 1 and 2, since 2 is about validation.